### DISCUSSION TITLE
Feature Request: Support for Animating 3D and Semi-Realistic AI-Generated Animal Portraits

### DISCUSSION THREAD

**Discussion by Gemglitch888** [KlingAIResearch/LivePortrait#409] [Ideas]
Could you please add a feature that allows us to animate 3D or semi-realistic pictures of animals? I'm unable to animate the AI-generated pictures I've created, as it says, "face not detected," but it works perfectly with real animal photos.

**Comment by cleardusk** [comment]
Does the animals mode not work for it?
